client/src/components/CategoryTile.js

```diff
@@ -2,9 +2,11 @@ import React from 'react'
 
 const CategoryTile = ({category}) => {
   return (
+    <div className="category-tile cell small-6">
       <h4>
         {category}
       </h4>
+    </div>
```

Comment:
If the `h4` is the only thing being returned we can remove the `div` and instead apply the class to the `h4`